doc/io.rst

```diff
@@ -338,6 +338,36 @@ supported by netCDF4-python: 'standard', 'gregorian', 'proleptic_gregorian' 'nol
 By default, xarray uses the 'proleptic_gregorian' calendar and units of the smallest time
 difference between values, with a reference time of the first time value.
 
+.. _io.iris:
+
+Iris
+----
+
+The Iris_ tool allows easy reading of common meteorological and climate model formats
+(including GRIB and UK MetOffice PP files) into ``Cube`` objects which are in many ways very
+similar to ``DataArray`` objects, while enforcing a CF-compliant data model. If iris is
+installed xarray can convert a ``Cube`` into a ``DataArray`` using
+:py:meth:`~xarray.Dataset.from_iris`:
+
+.. ipython:: python
+    :verbatim:
```

Comment:
`:verbatim:` is basically designed for copying/pasting output from the IPython command line.

I would definitely prefer adding Iris to our docs builds if that's feasible. See:
https://github.com/pydata/xarray/blob/master/doc/environment.yml

Reply:
OK, I'll give it a shot. How does the environment work, are all the examples run in the same interpreter session? And are there pre-built datarrays I can just use as examples?